db/migrate/20230308215104_add_raw_query_strings.rb

```diff
@@ -1,5 +1,5 @@
 class AddRawQueryStrings < ActiveRecord::Migration[7.0]
   def change
-    add_column :visits, :raw_query_string, :text
+    add_column "#{Land.config.schema}.visits", :raw_query_string, :text
```

Comment:
This was changed in billdoctor.

<img width="570" alt="Screenshot 2025-02-06 at 6 28 06 PM" src="https://github.com/user-attachments/assets/e115b525-51cd-4848-9de1-9fd44e60f00e" />